core/src/main/scala/cats/syntax/apply.scala

```diff
@@ -23,14 +23,17 @@ package cats
 package syntax
 
 trait ApplySyntax extends TupleSemigroupalSyntax {
-  implicit final def catsSyntaxApply[F[_], A](fa: F[A])(implicit F: Apply[F]): Apply.Ops[F, A] =
+  final def catsSyntaxApply[F[_], A](fa: F[A])(F: Apply[F]): Apply.Ops[F, A] =
```

Comment:
I guess we should deprecate it.

```suggestion
  @deprecated("Use `catsSyntaxApplyBinCompat1`", "2.10.0")
  final def catsSyntaxApply[F[_], A](fa: F[A], F: Apply[F]): Apply.Ops[F, A] =
```